docs/management/connectors/action-types/email.asciidoc

```diff
@@ -22,11 +21,21 @@ configuration setting is used, the email addresses used for all of the Sender
 configuration setting.
 ====
 
+[float]
+[[define-email-ui]]
+=== Create connectors in {kib}
+
+You can create connectors in *{stack-manage-app} > {connectors-ui}*
+or as needed when you're creating a rule. For example:
+
+[role="screenshot"]
+image::management/connectors/images/email-connector.png[Email connector]
```

Comment:
This screenshot is pretty out of date...are there plans to update it?

Reply:
I'm working my way through automating the screenshots and this can be one of the next pages I prioritize.